src/util.c

```diff
@@ -231,7 +231,7 @@ netplan_util_create_yaml_patch(const char* conf_obj_path, const char* obj_payloa
 
 // LCOV_EXCL_START
 file_error:
-    g_set_error(error, G_FILE_ERROR, errno, "Error when opening FD %d: %s", output_fd, g_strerror(errno));
+    g_set_error(error, NETPLAN_FILE_ERROR, errno, "Error when opening FD %d: %s", output_fd, g_strerror(errno));
```

Comment:
Should we use a NETPLAN_ERROR_ERRNO code here, to avoid conflicts?

Reply:
nitpick: `_OS_ERROR` would be better than `_ERRNO` IMHO